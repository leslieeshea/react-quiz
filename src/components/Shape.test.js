import React from 'react';
import { shallow } from 'enzyme';
import Shape from './Shape';

describe('Shape component', () => {
  it('renders a Shape', () => {
    const wrapper = shallow(<Shape text="hi" color="#FF0000" backgroundColor="#FF0000" />);
    expect(wrapper).toMatchSnapshot();
  });
});
