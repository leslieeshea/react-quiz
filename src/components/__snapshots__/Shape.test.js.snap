// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shape component renders a Shape 1`] = `
<div
  style={
    Object {
      "backgroundColor": "#FF0000",
      "color": "#FF0000",
      "height": "100px",
      "width": "100px",
    }
  }
>
  <p>
    hi
  </p>
</div>
`;
